src/main/java/uk/gov/justice/digital/delius/data/api/Enforcement.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Enforcement {
+    @NotNull
```

Comment:
maybe "required" should be force, it has no real impact but people might think it means there is always an enforcement

Reply:
EnforcementAction is required if there is an Enforcement, hence required true